### PR TITLE
add clipcontext tool yaml

### PR DESCRIPTION
**Pull request by michauhl**
Hi,
this recipce adds the clipcontext tool to bioconda. It's a Python-based tool, installed with setup.py install. Tests are available in the repo (both for library and executables) with detailed descriptions in the Github README.
Best,
Michael

----

> Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences**
      (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

> Everyone has access to the following BiocondaBot commands, which can be given in a comment:
>
>  * `@BiocondaBot please update` will cause the BiocondaBot to merge the master branch into a PR
>  * `@BiocondaBot please add label` will add the `please review & merge` label.
>  * `@BiocondaBot please fetch artifacts` will post links to packages and docker containers built by the CI system. You can use this to test packages locally before merging.
>
> For members of the Bioconda project, the following command is also available:
>
>  * `@BiocondaBot please merge` will cause packages/containers to be uploaded and a PR merged. Someone must approve a PR first! This has the benefit of not wasting CI build time required by manually merging PRs.
>
> If you have questions, please post them in gitter or ping `@bioconda/core` in a comment (if you are not able to directly ping `@bioconda/core` then the bot will repost your comment and enable pinging).

### CODE REVIEW

File: recipes/clipcontext/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  version: "0.1"
+
+source:
+  url: https://github.com/BackofenLab/CLIPcontext/archive/v0.1.tar.gz
+  sha256: df330517c397c8c3f1315fb020ff1dc611d923f9d1dcdb4de6fce3fda3c35c1b
+
+build:
+  number: 0
+  noarch: python
+  script: $PYTHON setup.py install --single-version-externally-managed --record=record.txt
```
Comment (bgruening) [inline recipes/clipcontext/meta.yaml]: can you please use pip install instead?

File: recipes/clipcontext/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+requirements:
+  host:
```
Comment (bgruening) [inline recipes/clipcontext/meta.yaml]: add pip here

File: recipes/clipcontext/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,34 @@
+package:
+  name: clipcontext
+  version: "0.1"
+
+source:
+  url: https://github.com/BackofenLab/CLIPcontext/archive/v0.1.tar.gz
```
Comment (bgruening) [inline recipes/clipcontext/meta.yaml]: use the `{{ version }}` syntax here and define this variable on top.

File: recipes/clipcontext/meta.yaml (hunk 1 above)
Comment (michauhl) [reply recipes/clipcontext/meta.yaml]: Uuh yes .. is the current one outdated?

Comment (michauhl) [reply recipes/clipcontext/meta.yaml]: Found this here:
  script: python -m pip install . --ignore-installed --no-deps -vv
Does this look okay?

Comment (bgruening) [reply recipes/clipcontext/meta.yaml]: jepp

File: recipes/clipcontext/meta.yaml [hunk 4]
```diff
@@ -1,13 +1,12 @@
...
 {% set sha256 = "df330517c397c8c3f1315fb020ff1dc611d923f9d1dcdb4de6fce3fda3c35c1b" %}
 
 package:
-  name: {{ name }}
+  name: clipcontext
   version: {{ version }}
 
 source:
-  url: https://github.com/BackofenLab/{{ name }}/archive/{{ version }}.tar.gz
+  url: https://github.com/BackofenLab/CLIPcontext/archive/{{ version }}.tar.gz
```
Comment (bgruening) [inline recipes/clipcontext/meta.yaml]: https://github.com/bioconda/bioconda-recipes/blob/736bcf9de1c75b21809695bdc5d3983eaaf710d1/recipes/pyensembl/meta.yaml#L6 :)

Comment (michauhl) [reply recipes/clipcontext/meta.yaml]: :grimacing: :smile_cat:

**Comment by michauhl** [comment]
Thanks for the help!

**Pull request merged**